src/nodes/model/linear/Dense.ts

```diff
@@ -12,22 +14,22 @@ export default class Dense extends Node {
 
     this.addOption('Size', 'IntegerOption', 1);
     this.addOption('Activation', 'DropdownOption', 'None', undefined, {
-      items: ['None', 'ReLU', 'Tanh', 'Sigmoid', 'Linear'],
+      items: valuesOf(BuiltinActivationF),
     });
     this.addOption('Use Bias', 'CheckboxOption', true);
 
     // TODO: Decide default value and options for these
     this.addOption('Weights Initializer', 'DropdownOption', 'Xavier', undefined, {
```

Comment:
I think we should also be using the enum for the default values. What do you think?

Reply:
I agree - maybe make it the first entry in the enum and take the first entry as default value? I am happy to make that a new PR, as I am eager to get this merged to rework IR

Reply:
I think the default value should be whatever the default value is in the framework we are targetting